app/services/rag_wise/chunking_service.rb

```diff
@@ -18,20 +18,34 @@ def file_chunking
              reader.pages.map(&:text).join(' ')
            when '.txt'
              File.read(@file.path)
+           when '.docx'
+             doc = Docx::Document.open(@file.path)
+             doc.paragraphs.map(&:text).join(' ')
+           when '.ipynb'
+             parse_ipynb(@file.path)
            else
              raise "Unsupported file type: #{@file_type}"
            end
 
     @text = text.gsub(/\s+/, ' ').strip
-    fixed_size_chunk_text(500, 100)
+    fixed_size_chunk_text(800, 100)
```

Comment:
Shouldn't you call `text_chunking` here, so you don't need to duplicate hardcode `(800, 100)` twice?

Reply:
Changed